app/views/components/_metadata.html.erb

```diff
@@ -1,31 +1,42 @@
 <%
-    published_at ||= ""
-    last_updated ||= ""
-    publishing_organisation ||= ""
-    document_type ||= ""
-    base_path ||= ""
-    status ||= ""
+    published_at ||= nil
+    last_updated ||= nil
+    publishing_organisation ||= nil
+    document_type ||= nil
+    base_path ||= nil
+    status ||= nil
 %>
 
 <div class="app-c-metadata govuk-body govuk-body-xs">
-  <% if status != "" %>
+  <% if status %>
     <dl class="app-c-metadata__list">
       <dt class="app-c-metadata__title"><%= t ".labels.status" %></dt>
       <dd class="app-c-metadata__description govuk-!-font-weight-bold"><%= status %></dd>
     </dl>
   <% end %>
   <dl class="app-c-metadata__list">
-    <dt class="app-c-metadata__title"><%= t ".labels.published_at" %></dt>
-    <dd class="app-c-metadata__description"><%= published_at %></dd>
-    <dt class="app-c-metadata__title"><%= t ".labels.last_updated" %></dt>
-    <dd class="app-c-metadata__description"><%= last_updated %></dd>
+    <% if published_at %>
+      <dt class="app-c-metadata__title"><%= t ".labels.published_at" %></dt>
+      <dd class="app-c-metadata__description"><%= published_at %></dd>
+    <% end %>
```

Comment:
I think everything must have published_at time, but I'm not totally confident...